src/renderers/WebGLRenderer.js

```diff
@@ -1461,6 +1462,9 @@ class WebGLRenderer {
 
 			_this.clear();
 
+			const renderBackground = xr.enabled === false || xr.isPresenting === false || xr.hasDepthSensing() === false;
```

Comment:
Do you mind passing the existing `renderBackground` flag from the `render()` method into `renderTransmissionPass()`. In this way, it's not necessary to make the same evaluation twice.

Reply:
Done!